### PR TITLE
TLC-38: Add Chemistry and Serology Lab Orders

### PR DESCRIPTION
**Pull request by ouiliam**
new file:   configuration/concepts/laborders.csv

	modified:   configuration/concepts/activate_forms.csv
	modified:   configuration/concepts/concepts.csv

### CODE REVIEW

**Comment by rbuisson** [comment]
Thanks. That seems good. Testing this on my local docker...

File: configuration/concepts/laborders.csv [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+,,,ASO Titer Data,,ASO Titer[<200],,,,Concept Details,N/A,,,ASO Titer; abn;,,,,,,,,
+,,,SLE Antibody Latex Test,,,,,,LabTest,Text,,,,,,,,,,,
+,,,Demo of Latex Test,,,,,,LabTest,Text,,,,,,,,,,,
+,,,Dimertest,,,,,,LabTest,Numeric,,,,TRUE,mg/l,,0.2,,,,
+,,,Dimertest Data,,Dimertest(plasma)[<0.2],,,,Concept Details,N/A,,,Dimertest; abn;,,,,,,,,
+,,,Dengue IgG,,,,,,LabTest,Coded,,Cambodia_Positive; Cambodia_Negative,,,,,,,,,
+,,,Dengue IgM,,,,,,LabTest,Coded,,Cambodia_Positive; Cambodia_Negative,,,,,,,,,
+,,,Pastorex Cryptococcus,,,,,,LabTest,Text,,,,,,,,,,,
+,,,TB Hexagon,,[Senior Dr only] TB Hexagon,,,,LabTest,Text,,,,,,,,,,,
+,,,Sodium,,,,,,ChemTest,Numeric,,,,TRUE,mmol/l,136,148,,,,
```
Comment (rbuisson) [inline configuration/concepts/laborders.csv:22]: Here you seem to use a Concept Class "ChemTest". Where does this concept class come from?

**Comment by rbuisson** [comment]
Problems while importing. I believe related to the ChemTest concept class. (See inline comments on the commits)
Have you tried this PR on a blank container?

**Comment by ouiliam** [comment]
In the Jira issue I explain that SQL commands again are needed for this (like with the age ranges)
https://mekomsolutions.atlassian.net/browse/TLC-38?jql=summary%20~%20%22lab*%22%20OR%20description%20~%20%22lab*%22%20ORDER%20BY%20lastViewed%20DESC
It worked on a blank container (after the SQL commands). You can also do the SQL commands after and it should work.

**Comment by rbuisson** [comment]
Thanks.

I have added the SQL commands needed to run as a SQL script available in the scripts/ folder.

**Pull request closed without merging**